src/less/components.less

```diff
@@ -24,7 +24,7 @@
 @import 'components/alerts';
 @import 'components/badges';
 @import 'components/box';
-@import 'components/breadcrumbs';
+@import 'components/breadcrumb/index.less';
```

Comment:
Remove the `.less` extension. LESS will automatically do this.

Reply:
`.less` extension removed.